default/web_tt2/suspend_request.tt2

```diff
@@ -1,114 +1,111 @@
+                                    [% additional_class = ' suspended' %]
+                                [% END %]
+                            [% END %]
+                        [% END %]
+                        <div class="item[% additional_class %]">
+                            <div class="item_content">
+                                <a class="item_title" href="[% 'info' | url_rel([l.key]) %]">
+                                    [%|obfuscate(conf.spam_protection) %][% l.key %][% END %]
+                                    [% IF l.value.admin %]
+                                        <span class="highlighted">([%|loc%]Owner[%END%])</span>
+                                    [% END %]
+                                </a>
+                                [% IF suspendable %]
+                                    [% IF suspended %]
+                                        [%|loc(sub.liststartdate)%]Suspended from %1 to[% END %]
+                                        [% IF sub.listenddate %]
+                                            [% sub.listenddate %]
+                                        [% ELSE %]
+                                            [%|loc%]indefinite end date[%END%]
+                                        [% END %]
+                                        <br/>
+                                        <input id="listname.[% sub.listname %]" type="checkbox" checked="checked" name="listname" value="[% l.key %]" />
+                                        <label for="listname.[% sub.listname %]">[%|loc(l.key)%]Check to restore reception from list %1[% END %]</label>
+                                    [% ELSE %]
+                                        <input id="listname.[% sub.listname %]" type="checkbox" checked="checked" name="listname" value="[% l.key %]" />
+                                        <label for="listname.[% sub.listname %]">[%|loc(l.key)%]Check to suspend or unsubscribe from list %1[% END %]</label>
+                                    [% END %]
+                                [% ELSE %]
+                                    <p class="note">
+                                        [%|loc%]You are not allowed to suspend your subscription / unsusbscribe from this list.[% END %]
+                                    </p>
+                                [% END %]
+                                <p class="list_subject">[% l.value.subject %]</p>
+                                <ul>
+                                    [% IF l.value.admin %]
+                                        <li>
+                                            <a href="[% 'review' | url_rel([l.key]) %]">
+                                                [%|loc%]Review members[%END%]
+                                            </a>
+                                        </li>
+                                    [% END %]
+                                    [% IF is_user_allowed_to('archive.web_access', l.key) %]
```

Comment:
This line should be:
``` code
                                    [% IF is_user_allowed_to('archive_web_access', l.key) %]
```
as change by 599aa22452c373d4f6c1c5f5742d638a1793fda9 in PR #559.

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de

Reply:
Confirmed.